Dockerfile.protofix

```diff
@@ -0,0 +1,21 @@
+FROM golang:latest
```

Comment:
This container does more than just protofix so it should be called proto I think.